torch/testing/_internal/opinfo/core.py

```diff
@@ -1762,7 +1762,7 @@ def generate_elementwise_binary_large_value_tensors(
 ):
     _large_int_vals = (-1113, 1113, -10701, 10701)
     _large_float16_vals = (-501, 501, -1001.2, 1001.2, -13437.7, 13437.7)
-    _large_float_vals = _large_float16_vals + (-4988429.2, 4988429.2, -1e20, 1e20)
+    _large_float_vals = _large_float16_vals + (-4988429.2, 4988429.2, -1e19, 1e19)
```

Comment:
These test sample filters seem really extreme considering the tests pass today without any of them. Perhaps you could investigate replacing the libc++ version we vendor with thrust's complex math library which has a permissive BSD license:
https://github.com/NVIDIA/thrust/tree/24486a169a62a58ef8f824d3dc9613c006b6f5a7/thrust/detail/complex

Reply:
@peterbell10 To confirm, You are suggesting to replace this implementation [llvm_complex.cpp](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/cuda/llvm_complex.cpp#L20) with thrust implementation?

Also Do you know of a good way to look at source code for generated JIT kernel, Currently I have to put `std::cout` in the execution path to see the source code. I wish to manually compile the generated source code using `nvcc`.